packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -5153,9 +5178,39 @@ export class WalletService {
     return sign;
   }
 
+
+  changellySignRequestsV2(message, secret: string) {
+    if (!message || !secret) throw new Error('Missing parameters to sign Changelly v2 request');
+
+    const privateKey = crypto.createPrivateKey({
+      key: secret,
+      format: 'der',
+      type: 'pkcs8',
+      encoding: 'hex'
```

Comment:
I don't think encoding is a valid param (anymore? maybe removed in node v18). For hex, it should be `key: Buffer.from(secret, 'hex'),`

Reply:
I changed it to use Buffer. Thank you!